lib/pymedphys/_utilities/test.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 University of New South Wales & Ingham Institute
```

Comment:
Hi @pchlap,

I think it's probably making the copyright of this one be in my name if that's okay.